### PR TITLE
Fix Death Saving Throws for dnd5e

### LINKED ISSUE #69: Death Saving Throws incorrectly roll "Death Savings Throw"
**Feedback Type**: Bug

**Core**: 0.7.9
**System**: dnd5e v1.2.4
**Module Version**: lmrtfy v1.11

At current, whenever a PC is called on to roll a Death Saving Throw, it incorrectly calls for a "Death Savings Throw" instead of a proper Death Saving Throw. (Picture Attached)

<p>Screen Shot 2021-03-13 at 4 26 05 PM
<br><i>The top roll is generated by LMRTFY, whereas the bottom is rolled using the character sheet.</i></p>

In addition, the attributes `attributes.death.success` and `attributes.death.failure` do not update properly.

### PR DESCRIPTION
**Pull request by shinkei13**
Resolves #69 

Proposed fix for https://github.com/League-of-Foundry-Developers/fvtt-module-lmrtfy/issues/69 by rolling Death Saves using actor.rollDeathSave similarly to how Initiative is handled with actor.rollInitiative.

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "lmrtfy",
     "title": "Let Me Roll That For You!",
     "description": "Ask your players to make rolls for you.",
-    "version": "1.9",
+    "version": "1.9.1",
```
Comment (akrigline) [inline module.json:6]: This change isn't necessary, our release automation handles this. We're actually at 1.11 right now and will hit 1.12 with this release (versioning scheme is just adopted from the original author's, ultimately the incrementing number is what matters)

**Review by akrigline — CHANGES_REQUESTED** [review]
Thanks for your contribution! Just a tiny tweak and I can push this out.

File: module.json (hunk 1 above)
Comment (shinkei13) [reply module.json:6]: Oh gotcha. I should have noticed that 1.11 was the version I had myself lol.

**Review by akrigline — APPROVED** [review]

**Comment by akrigline** [comment]
Thanks for pring this fix! Release will be pushed out today.

**Pull request merged**